src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/network/sdk/AzureVirtualNetworkOperationSDK.java

```diff
@@ -93,4 +103,50 @@ private void subscribeVirtualNetworkCreation(Observable<Indexable> virtualNetwor
                 .subscribe();
     }
 
+    public AzureGetVirtualNetworkRef doGetInstance(String resourceName, AzureUser azureUser)
+            throws FogbowException {
+
+        Network network = getNetwork(resourceName, azureUser);
+
+        VirtualNetworkInner virtualNetworkInner = network.inner();
+        String provisioningState = virtualNetworkInner.provisioningState();
+        String id = virtualNetworkInner.id();
+        String name = network.name();
```

Comment:
In the name variable, you must return the name of the network defined in the order and inserted in the resource tag.

Reply:
Done.
Note: I have fixed it in the last branch of Azure Network context.